ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -87,7 +87,12 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
 ))@
 
 @[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker python3 python3-catkin-pkg-modules python3-pip 
+# Focal abi-compliance-checker package has a bug that breaks python invocation
+# See: https://github.com/lvc/abi-compliance-checker/pull/80#issuecomment-652521014
+# Install 2.3 version from source, needs perl
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y curl make perl python3 python3-catkin-pkg-modules python3-pip
+RUN curl -sL https://github.com/lvc/abi-compliance-checker/archive/2.3.tar.gz | tar xvz -C /tmp
+RUN make install prefix=/usr -C /tmp/abi-compliance-checker-2.3
```

Comment:
Should the last two `RUN` lines be collapsed into a single line to avoid the first being cached but the second failing (due to an incorrect download)? The line could also delete `/tmp/abi-compliance-checker-2.3` afterwards to keep the image size smaller.

Reply:
sounds correct a18d87bc7b7ff7be56209ec4adbb7a99c7c04d85